pkg/controller/migmigration/validation.go

```diff
@@ -46,6 +48,8 @@ const (
 	RunningMessage           = "Step: %d/%d"
 	FailedMessage            = "The migration has failed.  See: Errors."
 	SucceededMessage         = "The migration has completed successfully."
+	UnhealthyDestination     = "The destination cluster has unhealthy migrated namespaces. See status.namespaces for details."
+	UnhealthySource          = "The source cluster has unhealthy namespaces. See status.namespaces for details."
```

Comment:
The `UnhealthySource` would be used by the plan controller and should be defined there.  Also, by convention, the _message_ const should end with `Message`.  Recommend: `UnhealthyNamespacesMessage` to match `UnhealthyNamespaces` condition type.